main.py

```diff
@@ -22,10 +22,24 @@ def question_name():
 def question_gender():
     print('Are you female or male?')
     gender = input('')
+    if gender == male or female:
+        start_story(name, gender)
+    else:
+        print('You must choose male or female')
     return gender
 
 def start_story(name, gender):
     pass
-name = question_name()
-gender = question_gender();
-start_story(name, gender)
+    if __name__ == '__main__':
```

Comment:
remove indent of if statement on line 33 and remove indentation from variables name and gender.